app/screens/ScheduleScreen/ScheduleDayPicker.tsx

```diff
@@ -51,33 +49,34 @@ AnimatedDayButton.displayName = "AnimatedDayButton"
 type ScheduleDayPickerProps = {
   scrollX: SharedValue<number>
   onItemPress: (itemIndex) => void
-  schedules: Schedule[]
-  selectedSchedule: Schedule
+  scheduleDates: Date[]
+  selectedScheduleDate: Date
```

Comment:
The expectation is that you're going to send me a `Date`. If you didn't, I would run into issues when I try to parse & format the date.